bin/varnishd/cache/cache_req_fsm.c

```diff
@@ -472,6 +472,11 @@ cnt_transmit(struct worker *wrk, struct req *req)
 
 	HSH_Cancel(wrk, req->objcore, boc);
 
+	if (!req->doclose && (req->objcore->flags & OC_F_FAILED))
+		/* The object we delivered failed due to a streaming error.
+		 * Fail the request. */
+		req->doclose = SC_REM_CLOSE;
```

Comment:
This sounds more like `RX_BAD` or `TX_ERROR`, definitely not "Client Closed".

Reply:
I just copied what the `v1d_vdp` would send if we caught this during transmission. No real opinion.

Reply:
In `v1d_vdp` it looks like a fallback.

Reply:
`SC_REM_CLOSE` does not have the `is_err` flag and also semantically it is definitely wrong.
From the existing errors, I would also vote for `SC_TX_ERROR`, but I wonder if we should have `SC_TX_SHORT`? If so, we would need to use it in at last one other place.

Reply:
Changed to `SC_TX_ERROR`